tests/integration/security/rbac/util/rbac_util.go

```diff
@@ -65,9 +70,15 @@ func (tc TestCase) CheckRBACRequest() error {
 			return getError(req, "allow with code 200", fmt.Sprintf("error: %v", err))
 		}
 	} else {
-		if req.Options.PortName == "tcp" {
-			if err == nil || !strings.Contains(err.Error(), "EOF") {
-				return getError(req, "deny with EOF error", fmt.Sprintf("error: %v", err))
+		if req.Options.PortName == "tcp" || req.Options.PortName == "grpc" {
+			expectedErrMsg := tcpDenyMsg
```

Comment:
Can we combine all the three cases, just to make the code cleaner? In http case, the expectedErrMsg is "deny with code 403".

Reply:
I think we can leave the code as it. The difference here is for http we expect err to be nil but 403 resp code. In gprc and tcp, we expect some err.